Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM alpine:3.6
+FROM alpine:latest
```

Comment:
We don't use `latest` because newer versions could have breaking changes.
Please, update for the specific version that you have tested.